### PR TITLE
add index for jobevent created

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

Add an index to `created` field on the `main_jobevent` table to make queries on events, specifically for analytics, faster.  

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - API

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by rooftopcellist** [comment]
recheck

File: awx/main/models/events.py [hunk 1]
```diff
@@ -204,6 +204,12 @@ class Meta:
         default=0,
         editable=False,
     )
+    created = models.DateTimeField(
```
Comment (ryanpetrello) [inline awx/main/models/events.py:207]: This is going to potentially be problematic (really slow) to add for people with loooooooooooooots of job event entries.

That said, we *do* need this for the analytics queries we want to do.

@matburt what say you.

Comment (matburt) [reply awx/main/models/events.py:207]: Yeah this alter table is going to be big, but I don't necessarily see a way around it. The index itself is going to be ~1GB per 100 million events.

Comment (matburt) [reply awx/main/models/events.py:207]: It might be worth mentioning this in the release notes, that the upgrade will take a little while.

Comment (ryanpetrello) [reply awx/main/models/events.py:207]: @rooftopcellist tried one of these on an `m3.large` w/ 120M rows and it took about 5 minutes.  I'm happy with this.

**Review by ryanpetrello — APPROVED** [review]

**Review by chrismeyersfsu — APPROVED** [review]

**Comment by rooftopcellist** [comment]
recheck

**Pull request merged**